src/mean.jl

```diff
@@ -24,11 +24,17 @@ julia> mean([:api00, :enroll], clus_one_stage)
    2 │ enroll  549.716  46.2597
 ```
 """
-function mean(x::Symbol, design::ReplicateDesign)
+function mean(x::Symbol, design::ReplicateDesign; ci_type::Union{Nothing,String}=nothing, kwargs...)
```

Comment:
Suggest using multiple dispatch. Make `mean(x::Symbol, design::ReplicateDesign; ci_type::Union{Nothing,String}=nothing, kwargs...)` that calls `mean(x::Symbol, design::ReplicateDesign)`

Reply:
ok

Reply:
this is the ways its being done now